VocaDbWeb/Startup.cs

```diff
@@ -292,6 +293,21 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 			app.UseAuthentication();
 			app.UseAuthorization();
 
+			// Code from: https://docs.microsoft.com/en-us/aspnet/core/security/anti-request-forgery?view=aspnetcore-6.0#generate-antiforgery-tokens-with-iantiforgery.
+			var antiforgery = app.ApplicationServices.GetRequiredService<IAntiforgery>();
+			app.Use((context, next) =>
+			{
+				var requestPath = context.Request.Path.Value;
+
+				if (string.Equals(requestPath, "/", StringComparison.OrdinalIgnoreCase) || string.Equals(requestPath, "/index.html", StringComparison.OrdinalIgnoreCase))
+				{
+					var tokenSet = antiforgery.GetAndStoreTokens(context);
+					context.Response.Cookies.Append("XSRF-TOKEN", tokenSet.RequestToken!, new CookieOptions { HttpOnly = false });

```

Comment:
Sets a cookie named `XSRF-TOKEN`. The client can read this cookie and provide its value as a header attached to AJAX requests. See [Generate antiforgery tokens with IAntiforgery](https://docs.microsoft.com/en-us/aspnet/core/security/anti-request-forgery?view=aspnetcore-6.0#generate-antiforgery-tokens-with-iantiforgery).